doc/releases/migration-guide-4.1.rst

```diff
@@ -110,3 +110,9 @@ Modem
 
 Architectures
 *************
+
+* x86
+
+  * Kconfigs ``CONFIG_DISABLE_SSBD`` and ``CONFIG_ENABLE_EXTENDED_IBRS`` have been deprecated
+    since v3.7. These are being removed.  Use :kconfig:option:`CONFIG_X86_DISABLE_SSBD` and
```

Comment:
Not a native speaker so maybe it's a nit, but feels better to say the following?

```suggestion
    since v3.7. These were removed.  Use :kconfig:option:`CONFIG_X86_DISABLE_SSBD` and
```

Reply:
Changed.